shell/common/switches.h

```diff
@@ -58,6 +58,10 @@ DEF_SWITCH(EndlessTraceBuffer,
            "This is useful when very old events need to viewed. For example, "
            "during application launch. Memory usage will continue to grow "
            "indefinitely however.")
+DEF_SWITCH(EnableSoftwareRendering,
+           "enable-software-rendering",
+           "Enable software rendering on the CPU. By default, Flutter will"
```

Comment:
"Enable rendering using the Skia software backend. This is useful when testing Flutter on emulators. By default, Flutter will attempt to either use OpenGL or Vulkan."